code/modules/mob/living/silicon/say.dm

```diff
@@ -69,10 +69,17 @@
 			message = trim(copytext(message,3))
 
 	//parse language key and consume it
+	var/needtohack = 1 //hacky way to get around stupid byond shit
 	var/datum/language/speaking = parse_language(message)
-	if (speaking)
+
+	if(speaking == null)
+		speaking = get_default_language()
+		needtohack = 0
+
+	if (speaking && !(speaking == null))
 		verb = speaking.speech_verb
-		message = trim(copytext(message,2+length(speaking.key)))
+		if(needtohack)

```

Comment:
Likewise here, if you move the language key removal before default language, you won't need the extra var.